examples/drawing/plot_random_geometric_graph.py

```diff
@@ -28,9 +28,9 @@
 
 plt.figure(figsize=(8, 8))
 nx.draw_networkx_edges(G, pos, nodelist=[ncenter], alpha=0.4)
-nx.draw_networkx_nodes(G, pos, nodelist=p.keys(),
+nx.draw_networkx_nodes(G, pos, nodelist=list(p.keys()),
```

Comment:
Did you get an error using ```p.keys()``` instead of ```list(p.keys())```?
As far as I can see from the code, an iterator with a length should be sufficient and ```p.keys()``` should provide this. I know the documentation says it should be a list, but perhaps it is better to change the docstring of ```draw_networkx_nodes``` than the code. But if you got an error using ```p.keys()``` then I need to investigate further.

Reply:
Yes, it's

```
WARNING: C:\Users\Admin\GIT\networkx\examples\drawing\plot_random_geometric_graph.py failed to execute correctly:Traceback (most recent call last):
  File "C:\Programming\Anaconda\envs\networkxsource\lib\site-packages\sphinx_gallery\gen_rst.py", line 450, in execute_code_block
    exec(code_block, example_globals)
  File "<string>", line 27, in <module>
  File "C:\Programming\Anaconda\envs\networkxsource\lib\site-packages\networkx-2.0rc1.dev20170828015156-py3.5.egg\networkx\drawing\nx_pylab.py", line 399, in draw_networkx_nodes
    label=label)
  File "C:\Programming\Anaconda\envs\networkxsource\lib\site-packages\matplotlib\__init__.py", line 1897, in inner
    return func(ax, *args, **kwargs)
  File "C:\Programming\Anaconda\envs\networkxsource\lib\site-packages\matplotlib\axes\_axes.py", line 3982, in scatter
    c_array = np.asanyarray(c, dtype=float)
  File "C:\Programming\Anaconda\envs\networkxsource\lib\site-packages\numpy\core\numeric.py", line 583, in asanyarray
    return array(a, dtype, copy=False, order=order, subok=True)
TypeError: float() argument must be a string or a number, not 'dict_values'
```

The problem is that a `KeysView` can't be used as iterable for `np.array`:

```
>>> np.array({1:3, 2:4}.keys())
array(dict_keys([1, 2]), dtype=object)
```

Reply:
I understand now --  Thank you.